examples/selenium/selenium-test.py

```diff
@@ -0,0 +1,17 @@
+from selenium import webdriver
```

Comment:
please add the copyright header boilerplate to this file (see hooks/boilerplate/...)